### PR TITLE
More Bug fixes to Chem Implant

### PR DESCRIPTION
**Pull request by CMOisLing**
Chem implant now behaves like a syringe would when injecting

### CODE REVIEW

**Comment by CMOisLing** [comment]
Not sure if I should have made a new branch but it doesn't look like my changes will affect anything else

**Comment by Sir-Lagsalot** [comment]
Wait, how the hell did they behave before?!

File: code/game/objects/items/implants/implant_chem.dm [hunk 1]
```diff
@@ -29,22 +29,20 @@
...
 	if(!cause || !imp_in)
 		return 0
 	var/mob/living/carbon/R = imp_in
 	var/injectamount = null
-	if (cause == "action_button")
+	if (cause == "action_button" || cause == "death")
 		injectamount = reagents.total_volume
 	else
 		injectamount = cause
+	reagents.reaction(R, INJECT, injectamount)
```
Comment (alexkar598) [inline code/game/objects/items/implants/implant_chem.dm]:
```suggestion
	reagents.reaction(R, INJECT)
```

thats a volume multiplier not a volume

**Review by alexkar598 — DISMISSED** [review]

**Comment by CMOisLing** [comment]
Wait, how the hell did they behave before?!

They would teleport the chems inside you and completely bypass any injection reactions. I found this out testing viruses.

**Pull request merged**